<?xml version="1.0"?>

<launch>
<arg name="view" value="false" />

   <node pkg="puzzle" type="pieces_info_pub" name="puzzle" output="screen">
        <param name="flip_image" value="false" />
        <param name="device_id" value="0" />
<!--        <param name="image_width" value="4672" />
        <param name="image_height" value="3504" /> -->
        <param name="image_width" value="640" />
        <param name="image_height" value="480" /> 

        <param name="frame_rate" value="1"/>
    </node>
    <group if="$(arg view)">

<!-- stuff that will only be evaluated if foo is true -->
        <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/cv_camera/image_raw"/>
        <param name="autosize" value="true" />
  </node>
  </group>
</launch>

